### ISSUE TITLE
cannot read entire code file when using JAWS or NVDA

### ISSUE THREAD

**Issue by jfayre** [microsoft/vscode#89717]
Issue Type: <b>Bug</b>

When using JAWS or NVDA screen readers to read through a file, I can only read down to line 200 or so. After that, the screen reader just says "blank" for every line. 
1. start JAWS or NVDA screen reader.
2. open a code file.
3. Using the arrow keys, navigate down past line 200 or so. screen reader will just say blank. 
I have attempted disabling extentions with no luck. 
This happened in a recent insider build. Stable build still works as expected.

VS Code version: Code - Insiders 1.42.0-insider (9e8c4c247a779cdd1f4f53db963bc6124a28ac2f, 2020-01-30T05:29:34.196Z)
OS version: Windows_NT x64 10.0.19551

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-7500U CPU @ 2.70GHz (4 x 2904)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>protected_video_decode: enabled<br>rasterization: enabled<br>skia_renderer: disabled_off<br>surface_control: disabled_off<br>surface_synchronization: enabled_on<br>video_decode: enabled<br>viz_display_compositor: enabled_on<br>viz_hit_test_surface_layer: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|7.89GB (1.67GB free)|
|Process Argv||
|Screen Reader|yes|
|VM|0%|
</details><details><summary>Extensions (3)</summary>

Extension|Author (truncated)|Version
---|---|---
python|ms-|2020.2.58744-dev
remote-wsl|ms-|0.42.1
vscodeintellicode|Vis|1.2.4

</details>

**Comment by isidorn** [comment]
@jfayre thanks for filling this issue. We have indeed changed something in this area: when we detect that a screen reader is detected we put 1000 lines in the textArea such that NVDA can read it. More details here
https://github.com/microsoft/vscode/commit/3efd575c64786100f67d80a29783715ccb3a9f18

I can reproduce this and for some weird reason NVDA can not handle textarea when there are more than 170 lines. I find this quite strange. However due to that I will change our strategy that we set the accessibilityPageSize to 160.
fyi @alexdima 

I have found the related NVDA issue https://github.com/nvaccess/nvda/issues/8781
